### PR TITLE
[YTAPI-5]Feature/create promotion

### PR DESCRIPTION
**Pull request by liptonday**
ticket: https://yourtable.atlassian.net/browse/YTAPI-5

### CODE REVIEW

File: api/swagger/swagger.yaml [hunk 1]
```diff
@@ -229,6 +229,27 @@ paths:
...
+          required: true
+        - name: upload
+          in: formData
+          type: file
+          required: true
+      responses:
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/ErrorResponse"
```
Comment (Smao0875) [inline api/swagger/swagger.yaml:252]: Should we also put a success code here?
 -- The future tests might need the success code.
"204":
          description: Success

**Review by Smao0875 — APPROVED** [review]
Everything seems fine.
 -- The success return code for PromotionController might not be neccessary if it is only used for admin test.

**Comment by liptonday** [comment]
@Smao0875 yes this is an admin endpoint. We probably need a UI for restaurant to import their data. But for now we go the easiest way :)

**Comment by Smao0875** [comment]
Got it.
I am modifying the code for deleting event.
Will let you know later today once I am done.

**Comment by liptonday** [comment]
👍 👍

**Pull request merged**